core/model/src/main/java/org/eclipse/rdf4j/IsolationLevel.java

```diff
@@ -22,7 +22,7 @@
 	/**
 	 * Shared constant for the {@link TransactionSetting} name used for isolation levels.
 	 */
-	static String NAME = IsolationLevel.class.getCanonicalName();
+	String NAME = IsolationLevel.class.getCanonicalName();
```

Comment:
o_0 why is this even a (public) constant?

Reply:
Probbaly for "performance" :P